### DISCUSSION TITLE
Welcome to LocoNetControlStation Discussions!

### DISCUSSION THREAD

**Discussion by positron96** [positron96/LocoNetControlStation#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZNIKSS** [comment]
Hello! Will this LocoNetControlStation work with TrainController ? Where can I communicate with you more comfortably?

**Reply by positron96** [reply]
Hi! Sorry I didn't notice your message before. Unfortunately, I have never worked with nor seen TrainController, so I really don't know. If it supports LocoNet command stations or can act as a WiThrottle client, then it might be possible.
